### ISSUE TITLE
GET Requests

### ISSUE THREAD

**Issue by h01d** [apavlidi/IT_API#51]
GET Requests should return 401 when access token expired instead of 400.

Access token error:
{"error":{"message":"An active access token required to complete this action.","type":"TokenInvalidError","code":400}}

**Comment by apavlidi** [comment]
@h01d Thanks for your feedback on this. Its an easy fix so I`ll annotate with good first issue for newcomers.

**Comment by iamaldi** [comment]
@apavlidi Can you verify that the following chunk of code is responsible for the issue reported?
https://github.com/apavlidi/IT_API/blob/5ec795c5613a5cb9464e048a797379ac585f4e8d/configs/auth.js#L53-L68

**Comment by apavlidi** [comment]
@iamaldi Yes this is it. Happy to hear you are interested in contributing 👍
